app/controllers/conference_controller.rb

```diff
@@ -5,7 +5,7 @@ class ConferenceController < ApplicationController
   load_resource :program, through: :conference, singleton: true, except: :index
 
   def index
-    @current = Conference.where('end_date >= ?', Date.current).order('start_date ASC')
+    @current = Conference.where('end_date >= ?', Date.current).reorder('start_date ASC')
```

Comment:
Yes, but it is the wrong order because sth goes wrong with ordering. Which is why I used reorder. It is still wrong with DESC.